apps/andi/lib/andi/input_schemas/options.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Andi.InputSchemas.Options do
+  @moduledoc false
+
+  def ratings() do
+    %{
```

Comment:
shouldn't these be defined in smartcitiesdata so they can be used everywhere as the same?

Reply:
I don't know of a requirement to do so *yet*.  Right now it's Andi's job to validate incoming data, and I'd like to keep it that way.  These ratings values are currently not exposed externally, so only Andi needs to know about them.  

We boy-scouted these other options into this file just to make things consistent, but I think we need to have a larger discussion about fields like 'language' and what the allowed values should be.